### PR TITLE
feat: added ability to disable routes

### PR DESCRIPTION
**Pull request by mprorock**

### CODE REVIEW

File: packages/vc-http-api/src/config/index.ts [hunk 1]
```diff
@@ -1,5 +1,12 @@
 const config = {
   version: '0.0.0',
+  routes: {
+    disabled: [
+      'disabled',
```
Comment (OR13) [inline packages/vc-http-api/src/config/index.ts:5]: as I read this, there are currently no disabled routes

**Review by OR13 — APPROVED** [review]
Looks good

File: packages/vc-http-api/src/config/index.ts (hunk 1 above)
Comment (mprorock) [reply packages/vc-http-api/src/config/index.ts:5]: correct - by default we leave everything on

Comment (mprorock) [reply packages/vc-http-api/src/config/index.ts:5]: with example on how to disable

**Pull request merged**